tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js

```diff
@@ -0,0 +1,136 @@
+    };
+
+    before(() => {
+        createdCloudStorageId = cy.attachS3Bucket(cloudStorageData);
+        cy.goToTaskList();
+        cy.openTaskJob(taskName);
+        cy.createRectangle(rectangleShape2Points);
+        cy.saveJob('PATCH', 200, 'saveJobExportDataset');
+        cy.goToTaskList();
+        cy.openTask(taskName);
+    });
+
+    after(() => {
+        cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
+        cy.url().should('include', '/cloudstorages');
+        cy.deleteCloudStorage(cloudStorageData.displayName);
+    });
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Export Annotation to the local storage', () => {
+            cy.clickInTaskMenu('Export task dataset', true, taskName);
+            cy.get('.cvat-modal-export-task').should('be.visible').find('.cvat-modal-export-select').click();
+            cy.contains('.cvat-modal-export-option-item', exportFormat).should('be.visible').click();
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-select').should('contain.text', exportFormat);
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-filename-input').type(annotationsArchiveNameLocal);
+            cy.contains('button', 'OK').click();
+            cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+            cy.closeNotification('.cvat-notification-notice-export-task-start');
+            cy.waitForDownload();
+        });
+
+        it('Export Annotation to the cloud storage', () => {
+            cy.clickInTaskMenu('Export task dataset', true, taskName);
+            cy.get('.cvat-modal-export-task').should('be.visible').find('.cvat-modal-export-select').click();
+            cy.contains('.cvat-modal-export-option-item', exportFormat).should('be.visible').click();
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-select').should('contain.text', exportFormat);
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-filename-input').type(annotationsArchiveNameCloud);
+            cy.get('.cvat-modal-export-task').find('.cvat-settings-switch').click();
+            cy.get('.cvat-select-target-storage').within(() => {
+                cy.get('.ant-select-selection-item').click();
+            });
+            cy.contains('.cvat-select-target-storage-location', 'Cloud storage').should('be.visible').click();
+
+            if (createdCloudStorageId) {
+                cy.get('.cvat-search-target-storage-cloud-storage-field').click();
+                cy.get('.cvat-cloud-storage-select-provider').click();
+            }
+            cy.contains('button', 'OK').click();
+            cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+            cy.closeNotification('.cvat-notification-notice-export-task-start');
+            cy.waitForFileUploadToCloudStorage();
+        });
+
+        it('Import Annotation from the local storage', () => {
```

Comment:
Also in the import datasets, you can use `uploadAnnotations` from [commands](https://github.com/opencv/cvat/blob/develop/tests/cypress/support/commands.js#L858)

Reply:
I tried using the `uploadAnnotations` but the problem is that `uploadAnnotations` is made assuming the annotation is uploaded from the jobs page. so the GET request is intercepted and checked it's status but if you upload annotation from the tasks page, there is no GET request. It only does POST and PUT requests. Hence I have not used `uploadAnnotations` but tried to write modularized code.